docs/docs/getting_started/quick_start.md

```diff
@@ -0,0 +1,115 @@
+
+:::
+
+## Nargo
+
+Nargo provides the ability to initiate and execute Noir projects. Let's initialize the traditional `hello_world`:
+
+```sh
+nargo new hello_world
+```
+
+Two files will be created.
+
+- `src/main.nr` contains a simple boilerplate circuit
+- `Nargo.toml` contains environmental options, such as name, author, dependencies, and others.
+
+Glancing at _main.nr_ , we can see that inputs in Noir are private by default, but can be labeled public using the keyword `pub`. This means that we will _assert_ that we know a value `x` which is different from `y` without revealing `x`:
+
+```rust
+fn main(x : Field, y : pub Field) {
+    assert(x != y);
+}
+```
+
+To learn more about private and public values, check the [Data Types](./noir/concepts/data_types/index.md) section.
+
+### Compiling and executing
+
+We can now use `nargo` to generate a _Prover.toml_ file, where our input values will be specified:
+
+```sh
+nargo check
+```
+
+Let's feed some valid values into this file:
+
+```toml
+x = "1"
+y = "2"
+```
+
+We're now ready to compile and execute our Noir program. By default the `nargo execute` command will do both, and generate the `witness` that we need to feed to our proving backend:
+
+```sh
+nargo execute hello-witness
+```
+
+You can now see the witness and the compiled circuit in the `target` folder. We're ready to prove
+
+## Barretenberg
+
+Using the `bb` CLI tool as an example:
+
+```sh
+bb prove -b ./target/hello_world.json -w ./target/hello-witness.gz -o ./target/proof
+```
+
+:::tip
+
+Naming can be confusing, specially as you pass them to the `bb` commands. If unsure, it won't hurt to delete the target folder and start anew to make sure you're using the most recent versions of the compiled circuit and witness.
+
+:::
+
+The proof is now generated in the `target` folder. To verify it we first need to compute the verification key from the compiled circuit, and use it to verify:
+
+```sh
+bb write_vk -b ./target/hello_world.json -o ./target/vk
+bb verify -k ./target/vk -p ./target/proof
+```
+
+:::info
+
+Notice that in order to verify a proof, the verifier knows nothing but the circuit, which is compiled and used to generate the verification key. This is obviously quite important: private inputs remain private.
+
+As for the public inputs, you may have noticed they haven't been specified. This behavior varies with each particular backend, but barretenberg typically attaches them to the proof. You can see them by parsing and splitting it. For example for if your public inputs are 32 bytes:
+
+```bash
+head -c 32 ./target/proof | od -An -v -t x1 | tr -d $' \n'
+```
```

Comment:
```suggestion
```

More appropriate to highlight in the "Private & Public Types" section of the "Data Types" page than in "Quick Start"

Reply:
I made it an admonition as to try to avoid the distraction, but I'd rather have a specific how-to guide on this than bury it in the concepts section. 

This is one of the most asked questions ever since nargo stopped proving and outputting the proof in hex format, so I'll leave it as visible as possible for the time being

Reply:
Feel free to reference this Issue if helpful: https://github.com/AztecProtocol/aztec-packages/issues/6487